src/util/rebind.js

```diff
@@ -24,4 +24,19 @@
         return target;
     };
 
+    function capitalizeFirstLetter(str) {
+        return str.charAt(0).toUpperCase() + str.slice(1);
+    }
+
+    /**
+     * Rebinds multiple properties from the source component, adding the given prefix.
+     */
+    fc.util.rebindProperties = function(target, source, properties, prefix) {
```

Comment:
How about calling it `rebindAll`, dropping `properties` and treating the arguments after `prefix` as excludes?